rtloader/common/builtins/datadog_agent.c

```diff
@@ -411,6 +419,41 @@ static PyObject *log_message(PyObject *self, PyObject *args)
     Py_RETURN_NONE;
 }
 
+/*! \fn PyObject *send_log(PyObject *self, PyObject *args)
+    \brief This function implements the `datadog_agent.send_log` method, sending
+    a log for eventual submission.
+    \param self A PyObject* pointer to the `datadog_agent` module.
+    \param args A PyObject* pointer to a 2-ary tuple containing a log line and the
+    unique ID of a check instance.
+    \return A PyObject* pointer to `None`.
+
+    This function is callable as the `datadog_agent.send_log` Python method and
+    uses the `cb_send_log()` callback to retrieve the value from the agent
+    with CGO. If the callback has not been set `None` will be returned.
```

Comment:
> If the callback has not been set `None` will be returned.

This seems to imply that a different result would be returned if a callback was set, which isn't the case

Reply:
Every function has this documentation; are they all incorrect?

Reply:
I don't think it's incorrect, but the last sentence seems misleading since `None` is always returned.
Now to be clear, this is definitely in the nit picking realm and can be ignored